src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -304,6 +306,17 @@ public static IResourceBuilder<PostgresServerResource> WithPgWeb(this IResourceB
                     Directory.CreateDirectory(serverFileMount.Source!);
                 }
 
+                // Need to grant read access to the config file on unix like systems.
```

Comment:
Why wasn't this needed before?

Reply:
Not sure how our machines or docker is setup. 

@danegsta ideas?

Reply:
The need for custom file permissions on Mac/Linux really depends on the specific container. If the container is running as root, it’s not really necessary, but if a container is running as a custom user, then we have to jump through hoops to make sure that user will be able to read the mounted file. So if the pgweb image started using a specific user instead of root, that’d necessitate the change.

Reply:
Its strange we dont need this on our other machine though right? I had to add it here because I saw permission errors

Reply:
I know this is per container, but I wonder if we can as part of creating a BindMount if we should allow setting these flags..

Reply:
Take a look at the new changes I made WRT bind mounts in the latest commits. I made it first class on bind mount and calls to `WithBindMount` can opt into what per resource.

I think we solve this on an integration by integration basis depending on what minimum permissions that bind mount needs based on the interaction with the container. I still don't know why I had to make these changes in this pr to make it work though.